zerver/views/webhooks/greenhouse.py

```diff
@@ -0,0 +1,61 @@
+from __future__ import absolute_import
+
+from django.utils.translation import ugettext as _
+from django.http import HttpRequest, HttpResponse
+from typing import Any
+
+from zerver.lib.actions import check_send_message
+from zerver.lib.response import json_success, json_error
+from zerver.decorator import REQ, has_request_variables, api_key_only_webhook_view
+from zerver.models import UserProfile, Client
+
+import ujson
+
+def message_creator(action, application):
+    # type: (str, Dict[str, Any]) -> str
+    def list_format(some_list):
+        # type: (List[Any]) -> str
+        internal_template = ''
+        for item in some_list:
+            try:
```

Comment:
Am I right that the basic idea of the try except block was getting `item['value']` or `item['url']` or add nothing?

If so you can do it like:
```
some_variable = item.get('value', item.get('url'))
if some_variable is not None:
    iternal_template ......
```